From 80c63801ef85562f2e2a855cd8dc07073110cd9c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:17 -0400
Subject: [usb] ftdi_sio: Add extra PID

Message-id: <1410887463-20674-546-git-send-email-dzickus@redhat.com>
Patchwork-id: 95629
O-Subject: [RHEL7 PATCH 545/591] USB: ftdi_sio: Add extra PID.
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 5a7fbe7e9ea0b1b9d7ffdba64db1faa3a259164c
Author: Bert Vermeulen <bert@biot.com>
Date:   Tue Jul 8 14:42:23 2014 +0200

    USB: ftdi_sio: Add extra PID.

    This patch adds PID 0x0003 to the VID 0x128d (Testo). At least the
    Testo 435-4 uses this, likely other gear as well.

    Signed-off-by: Bert Vermeulen <bert@biot.com>
    Cc: Johan Hovold <johan@kernel.org>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/ftdi_sio.c b/drivers/usb/serial/ftdi_sio.c
index f46199e..884147f 100644
--- a/drivers/usb/serial/ftdi_sio.c
+++ b/drivers/usb/serial/ftdi_sio.c
@@ -717,7 +717,8 @@ static const struct usb_device_id id_table_combined[] = {
  { USB_DEVICE(FTDI_VID, FTDI_ACG_HFDUAL_PID) },
  { USB_DEVICE(FTDI_VID, FTDI_YEI_SERVOCENTER31_PID) },
  { USB_DEVICE(FTDI_VID, FTDI_THORLABS_PID) },
- { USB_DEVICE(TESTO_VID, TESTO_USB_INTERFACE_PID) },
+ { USB_DEVICE(TESTO_VID, TESTO_1_PID) },
+ { USB_DEVICE(TESTO_VID, TESTO_3_PID) },
  { USB_DEVICE(FTDI_VID, FTDI_GAMMA_SCOUT_PID) },
  { USB_DEVICE(FTDI_VID, FTDI_TACTRIX_OPENPORT_13M_PID) },
  { USB_DEVICE(FTDI_VID, FTDI_TACTRIX_OPENPORT_13S_PID) },
diff --git a/drivers/usb/serial/ftdi_sio_ids.h b/drivers/usb/serial/ftdi_sio_ids.h
index a387244..4b437b2 100644
--- a/drivers/usb/serial/ftdi_sio_ids.h
+++ b/drivers/usb/serial/ftdi_sio_ids.h
@@ -791,7 +791,8 @@
  * Submitted by Colin Leroy
  */
 #define TESTO_VID   0x128D
-#define TESTO_USB_INTERFACE_PID  0x0001
+#define TESTO_1_PID   0x0001
+#define TESTO_3_PID   0x0003
 
 /*
  * Mobility Electronics products.
-- 
1.7.1